### DISCUSSION TITLE
Sudden audio baseline change

### DISCUSSION THREAD

**Discussion by chsungyesuzuki** [ppy/osu#14628] [Q&A]
**Describe the bug:** As the title said, there are sudden lag increase and audio baseline change after almost no operations (i.e. downloading 2 beatmapsets and importing them and editing a new beatmap and quitting the edit without saving). Before my offset is approx. +30ms and now I have to set it to +130ms. And when playing, my keyboard input lag significantly increases.

**osu!lazer version:** 2021.828.0

**Logs:** I don't know which log should be attached here and the log shows nothing happened that time.
[database.log](https://github.com/ppy/osu/files/7109728/database.log)
[network.log](https://github.com/ppy/osu/files/7109730/network.log)
[performance.log](https://github.com/ppy/osu/files/7109731/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7109732/runtime.log)

**Comment by bdach** [comment]
Please attach a video of this if you can, preferably with the extended frame counter display showing (press <kbd>Ctrl</kbd>+<kbd>F11</kbd> twice).

**Comment by chsungyesuzuki** [comment]
Sorry. It now feels that the lag is an illusion due to the audio problem, but the change in audio baseline is real.

[before.mp4](https://user-images.githubusercontent.com/35860604/132095371-e9393767-d9cb-4e68-ae8b-4e96edf1cc9b.mp4)


[after.mp4](https://user-images.githubusercontent.com/35860604/132095375-28d0884e-e390-4eda-866a-7c5231370792.mp4)

**Reply by smoogipoo** [reply]
What are the exact steps to reproducing this issue? Can you list it out in a chronological order? From the issue description it sounds like play map -> exit -> download/import 2 new maps -> play map again.

**Comment by chsungyesuzuki** [comment]
Though the videos seem similar, the audio offset is significantly different.

**Comment by peppy** [comment]
Moving to discussion as it's likely something specific to system configuration. Welcome to convert back to an issue if we come across anything actionable.
